.github/workflows/main.yml

```diff
@@ -69,32 +73,70 @@ jobs:
       run: |
         python3 -m pip install networkx defusedxml test-generator===0.1.1
         source /opt/intel/openvino_2021/bin/setupvars.sh
+        export MO_DIR=/opt/intel/openvino_2021/deployment_tools/model_optimizer/
 
         versions=(d0 d4)
         sizes=(512 1024)
         for (( i=0; i<2; i++ )); do
           version=${versions[$i]}
           size=${sizes[$i]}
 
-          python3 openvino/model-optimizer/mo.py \
+          python3 $MO_DIR/mo.py \
             --input_model automl/efficientdet/savedmodeldir-${version}/efficientdet-${version}_frozen.pb \
-            --transformations_config openvino/model-optimizer/extensions/front/tf/automl_efficientdet.json \
+            --transformations_config $MO_DIR/extensions/front/tf/automl_efficientdet.json \
             --input_shape "[1, ${size}, ${size}, 3]" \
-            --model_name efficientdet-${version}
+            --model_name efficientdet-${version} \
+            --output_dir efficientdet-${version}-FP32 --data_type FP32
 
-          python3 scripts/validate.py --version ${version} --width ${size} --height ${size}
+          python3 $MO_DIR/mo.py \
+            --input_model automl/efficientdet/savedmodeldir-${version}/efficientdet-${version}_frozen.pb \
+            --transformations_config $MO_DIR/extensions/front/tf/automl_efficientdet.json \
+            --input_shape "[1, ${size}, ${size}, 3]" \
+            --model_name efficientdet-${version} \
+            --output_dir efficientdet-${version}-FP16 --data_type FP16
         done
 
+    - name: Validate IR
+      run: |
+        source /opt/intel/openvino_2021/bin/setupvars.sh
+
+        versions=(d0 d4)
+        sizes=(512 1024)
+        for (( i=0; i<2; i++ )); do
+          version=${versions[$i]}
+          size=${sizes[$i]}
+
+          echo "-------- Validate efficientdet-${version} FP32 IR --------"
+          ln -sf efficientdet-${version}-FP32/efficientdet-${version}.bin ./efficientdet-${version}.bin
+          ln -sf efficientdet-${version}-FP32/efficientdet-${version}.xml ./efficientdet-${version}.xml
+          python3 scripts/validate.py --version ${version} --width ${size} --height ${size} 2>&1 | grep -iv cuda
```

Comment:
Why output is redirected?

Reply:
Tensorflow cuda warnings are written to `stderr` and are not really useful since GitHub actions doesn't give us cuda anyway.. This will ensure that the `validate.py` script only writes what you intended it to write.